frontend/src/global_styles/common/select/select.sass

```diff
@@ -18,3 +18,18 @@
     &:hover
       color: var(--list-item-hover--color)
       background-color: var(--control-transparent-bgColor-hover)
+
+
+select
+  -webkit-appearance: none !important
+  -moz-appearance: none !important
+  background: var(--body-background) !important
+  background-image: var(--select-arrow-bg-color-url) !important
+  background-repeat: no-repeat !important
+  background-position-x: 100% !important
+  background-position-y: 5px !important
+  margin-right: 2rem !important
+  padding-right: 2rem !important
+
+  &>option
+    background-color: var(--body-background)
```

Comment:
Please double check whether all of this (and the new variable) is really needed. We already have a similar thing [here](https://github.com/opf/openproject/blob/dev/frontend/src/global_styles/openproject/_forms.sass#L160). Ideally, you'd only have to change the `$select-arrow-color`.

Reply:
Hi @HDinger,
thanks for the hints, [image-triangle() function](https://github.com/opf/openproject/blob/1f3c0d72fa4c7777e0c5573df482172ee260732c/frontend/src/global_styles/openproject/_functions.sass#L56) only receives colors like '#000', we can't pass a variable to it and $select-arrow-color is not a global variable, so I can't map it to another value in primer-mapping file.